src/main/java/com/dawn/jat/illuminati/post/controller/PostController.java

```diff
@@ -38,4 +41,26 @@
         }
         return new ResponseEntity<>(postEntity, HttpStatus.OK);
     }
-}
+
+    /**
+     * Gets list of Post Brief.
+     */
+    @GetMapping(value = "/list-post-brief")
+    public ResponseEntity<Object> getListPostBrief() {
+        List<PostEntity> allPost = postService.findAll();
```

Comment:
We dont need the whole PostEntity here
Change `findAll` to new method `findPostSummary` 
There are 2 solution:
1. Create a new Entity PostBrief, or PostSummary contained only necessary information, it's still connect to collection post. PostService will call PostSummaryRepository to query all post summary
2. Still using the PostRepository, but filter out only necessary information and return back

Reply:
Updated